force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -136,6 +136,13 @@
         <shortDescription>Review Step of the Bulk Service Delivery Tool</shortDescription>
         <value>Review</value>
     </labels>
+    <labels>
+        <fullName>BSDT_Warning_No_Contacts</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning when no contacts are selected for the BSDT Group Option</shortDescription>
+        <value>Please select at least one contact to continue.</value>
```

Comment:
@jjbennett , one quick question for you. Where would we see this message? On the Contact Selection we cannot click on the next button without selecting a contact.

Reply:
@bmuniswamy Great question, it's actually on the hover of the disabled next button. So if they are like why can't I click Next and they hover they will find out why :)